File: cfgs/java_cfgs_foo/MAXIMUM_SUM_PAIRS_SPECIFIC_DIFFERENCE_1.java
Nodes
1;special;;;
2;statement;int maxSum = 0;14;
3;statement;fun_1(foo_1);15;
4;statement;int i = foo_1 - 1;16;
5;conditional;foo_1 > 0;17;
6;conditional;(foo_1[foo_2] - foo_3[foo_4 - 1]) < foo_5;19;
7;statement;maxSum += foo_1[foo_2];20;
8;statement;maxSum += foo_1[foo_2 - 1];21;
9;statement;--i;22;
10;statement;--i;18;
11;statement;return foo_1;25;
12;exit;;;
Edges
0;2;
1;3;
2;4;
3;5;
4;6;11;
5;7;10;
6;8;
7;9;
8;10;
9;5;
10;12;
11;

File: cfgs/pythoncfgs/MAXIMUM_SUM_PAIRS_SPECIFIC_DIFFERENCE_1.py
Nodes
1;statement;foo_1 = 0;8;
2;statement;fun_1();9;
3;statement;foo_1 = foo_2 - 1;10;
4;conditional;foo_1 >= 0;11;
5;statement;return foo_1;17;
6;conditional;foo_1[foo_2] - foo_3[foo_4 - 1] < foo_5;12;
7;statement;foo_1 -= 1;16;
8;statement;foo_1 += foo_2[foo_3];13;
9;statement;foo_1 += foo_2[foo_3 - 1];14;
10;statement;foo_1 -= 1;15;
11;exit;;
Edges
0;2;
1;3;
2;4;
3;5;6;
4;11;
5;7;8;
6;4;
7;9;
8;10;
9;7;
10;